src/screens/RequestSwap/SwapSummary.tsx

```diff
@@ -16,16 +16,30 @@ export const SwapSummary = (props) => {
     const errorState = useSelector(
         (state) => state.loadStateReducer.loadErrorStatus.SWAP,
     );
-    const { option, date, target } = useSelector((state) => state.swapReducer);
-
-    const onConfirmPress = () => {
-        dispatch(sendSwapRequest(option, date, target));
-    };
+    const { myTask, targetTask } = useSelector((state) => state.swapReducer);
 
```

Comment:
Added a default message and an input field for custom messages. Subject to change

Btw I don't think we discussed custom messages attached to the notifications. Yea or Nay?

Reply:
We can just leave it until we run through it and see if we like it better with custom messages or not